README.md

```diff
@@ -309,3 +309,31 @@ import pymongo
 client = pymongo.MongoClient("connection string")
 client.list_database_names()
 ```
+
+## Setting Up GitHub OAuth
+
+After setting up the application in GitHub, you need to add the details to this section of the .env file:
+
+```
+Copy code
+OAUTH_ID=
+OAUTH_KEY=
+OAUTH_URL=
+ENV=
```

Comment:
I'm not sure I follow why the ENV variable is required, if the other three variables let you vary the values between the local and deployed versions?

Reply:
Hello, the primary reason for this issue stems from the redirect URL utilized in the GitHub OAuth process. When running the application locally, the OAuth flow must redirect to http://127.0.0.1:5000 instead of the Azure-hosted app's URL. Consequently, I was compelled to register distinct applications on GitHub.

Reply:
Separate apps in GitHub definitely makes sense, but can we just vary the ID/Key/URL in different environments to use the right one?